src/App.js

```diff
@@ -23,29 +23,26 @@ function App() {
   };
 
   const [isModalOpen, setIsModalOpen] = useState(false);
-  const [activeMentorId, setActiveMentorId] = useState(null);
+  const [activeMentorId, setActiveMentorId] = useState(0);
```

Comment:
I wonder if there's a better solution to this than to set a default value of 0, since the modal would fail if no mentors are provided. But this is a pretty minor edge case and I doubt it'll be of much significance.